### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by phoebeww**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,163 @@
...
+
+### Building and Running a Local Instance
+In order to build and use the service of the project, you must install the following:
+
+1. [Maven 3.9.5](https://maven.apache.org/download.cgi) Download and follow the installation instructions. Set the bin as a new path variable for both Windows and MacOS.
+2. [JDK 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) used for development.
+3. [IntelliJ IDE](https://www.jetbrains.com/idea/download/?section=windows) Or use other IDE of your preference.
+
+In order to build the project, run under `./100of100Service` directory:
+```bash
```
Comment (Alex-XJK) [inline README.md:92]: Please double-check this command!

**Review by Alex-XJK — CHANGES_REQUESTED** [review]
In general it looks good to me! 
But please make sure the highlighted command again. Thanks!

**Review by sophiefeifeifeiya — APPROVED** [review]
/lgmt

**Review by Alex-XJK — APPROVED** [review]
Looks good to me, thank you so much!

**Pull request merged**